### ISSUE TITLE
A union named 'data' causes the build to fail

### ISSUE THREAD

**Issue by rocallahan** [capnproto/capnproto-rust#77]
I have a schema containing a struct field named `data` that's a union. That causes a build failure:
````
error[E0255]: a module named `data` has already been imported in this module
    --> /home/roc/pernosco/code/target/debug/build/rr-trace-3d4d5d2e72ce7323/out/rr_trace_capnp.rs:3240:7
     |
3064 |         use capnp::{text, data, Result};
     |                           ---- previous import of `data` here
...
3240 | /       pub mod data {
3241 | |         #![allow(unused_imports)]
3242 | |         use capnp::capability::{FromClientHook, FromTypelessPipeline};
3243 | |         use capnp::{text, data, Result};
...    |
3458 | |         pub type WhichBuilder<'a,> = Which<Result<primitive_list::Builder<'a,i32>>,Result<struct_list::Builder<'a,::rr_trace_capnp::opened_fd::Owned>>>;
3459 | |       }
     | |_______^ `data` already imported
````

**Comment by rocallahan** [comment]
I'm working around this by changing my schema but it's a bit sad.

**Comment by dwrensha** [comment]
Thanks for the report!

This is fixed by https://github.com/capnproto/capnpc-rust/commit/a3a5b53f5f173690686f2296d1e107b03d0ba01f, which is included in the new capnpc v0.8.7 release.

**Comment by rocallahan** [comment]
Fantastic thanks!
